pom.xml

```diff
@@ -117,7 +117,7 @@
 
     <!-- use the same values in bom/pom.xml -->
     <zipkin.version>2.12.5</zipkin.version>
```

Comment:
might as well update zipkin also, as we were waiting on that also..

Reply:
to match reporter would be 2.14.0.. PS there's probably some find/replacing to do here and elsewhere notably around group ID switches. I use the `ag` tool like.. `ag io.zipkin` to find references.